cli/command/registry.go

```diff
@@ -28,16 +27,22 @@ const (
 		"for organizations using SSO. Learn more at https://docs.docker.com/go/access-tokens/"
 )
 
+// authConfigKey is the key used to store credentials for Docker Hub. It is
+// a copy of [registry.IndexServer].
+//
+// [registry.IndexServer]: https://pkg.go.dev/github.com/docker/docker/registry#IndexServer
+const authConfigKey = "https:/index.docker.io/v1/"
```

Comment:
A drive-by comment: This should have `//` in `https://`.

Reply:
OMG! 🙈 Would that be it??


I was looking over the code multiple times to see where I went wrong, and this ... definitely could be it!

I'm not sure we ever met in person, but if we do (I hope we will, one day), I owe you a 🍻 (or any other beverage) ❤️